meson.build

```diff
@@ -30,18 +30,101 @@ add_project_arguments(cxx.get_supported_arguments(extra_flags), language: 'cpp')
 
 conf = configuration_data()
 conf.set_quoted('PACKAGE_VERSION', meson.project_version())
-conf.set_quoted('DATADIR', join_paths(get_option('prefix'), get_option('datadir')))
+conf.set_quoted('NP_DATADIR', join_paths(get_option('prefix'), get_option('datadir')))
 config_h = configure_file(output : 'config.h', configuration : conf)
 
 meson.add_devenv({'NETPANZER_DATADIR': meson.project_source_root()})
 
-deps = [
-  dependency('SDL2'),
-# SDL2 is case sensitive for the ttf library
-  dependency('SDL2_ttf'),
-  dependency('SDL2_mixer'),
-  dependency('physfs')
-]
+deps = []
+
+if host_machine.system() == 'windows'
+    deps += cxx.find_library('ws2_32')
+endif
+
+# this won't use fallback defined in subprojects/lua.wrap
+lua_dep = dependency('lua-5.1', required: false)
+if not lua_dep.found()
+  lua_dep = cc.find_library('lua', has_headers: 'lua.h', required: false)
+  if not lua_dep.found()
+    # This will use the fallback
+    lua_dep = dependency('lua-5.1')
+    # or
+    lua_dep = dependency(
+      'lua-5.1',
+      required: false,
+      fallback: 'lua',
+      default_options: ['default_library=static']
+      )
```

Comment:
There's no reason to look this up twice, especially, the options specified the second time won't be used as the first lookup has to succeed.